### ISSUE TITLE
Stay connected button

### ISSUE THREAD

**Issue by AnOnyme77** [turtl/desktop#87]
Hello, is it possible to have a "stay connected button" on desktop app (Mac in my case) ? It's really annoying to connect each time I launch the app.

**Comment by rubbingalcoholic** [comment]
But wouldn't that make it much less secure? Anyone who gained physical access to your computer could instantly pwn your account. Wouldn't it be more secure to use a password manager like 1password?

**Comment by AnOnyme77** [comment]
I agree with that but this button exists in Android App so why not in the desktop one ?

**Comment by mire3212** [comment]
@rubbingalcoholic I agree that once physical access is gained it makes the notes immediately accessible, but if someone gains physical access to a system then there are a lot of other worse things they can do to extract data. 

For instance, a backdoor to allow persistent access to the machine
A keylogger to capture passwords, including those from Turtl
A root kit to allow persistent access, remote screen viewing, etc.

Offering an option to stay logged in would allow the user to make that choice between security and convenience.

This is the one feature that has prevented me from moving to Turtl as a more permanent solution as I don't want to have to type a 16+ random password every time I want to access my notes. This means I leave Turtl open on the screen in order to have quick access. This means that if someone gets physical access, Turtl is already open and they would have the same access anyway.

**Comment by liquidsunshine** [comment]
The need to pull up a password manager every time I launch the app basically makes it unusable.

I have an encrypted hard drive, keep my machine fully patched, and I lock it when it's not in use.  I mostly want the encryption because it keeps my data safe while it's "in the cloud", and this would not at all be degraded by leaving the app logged in at all times on my local computer.  

For people who have a weaker threat model than me, they could just leave that box unchecked.
